doc/setup/install.rst

```diff
@@ -564,3 +575,4 @@ default db to serve on localhost:8069
 .. _Editions: https://www.odoo.com/pricing#pricing_table_features
 .. _nightly: https://nightly.odoo.com/10.0/nightly/
 .. _extra: https://nightly.odoo.com/extra/
+.. _Odoo SH: https://www.odoo.sh/
```

Comment:
Do we write `Odoo SH` like that in other places? Shouldn't it be `Odoo.sh`?

Reply:
dots don't work here :p 
didn't find a way to escape 'em  ^^

Reply:
I couldn't find how to escape it either. Get rid of the variable then and just add the link once with the normal link syntax?